pom.xml

```
@@ -262,7 +262,7 @@
         <gravitee-policy-javascript.version>1.3.3</gravitee-policy-javascript.version>
         <gravitee-resource-auth-provider-http.version>1.4.0</gravitee-resource-auth-provider-http.version>
         <gravitee-resource-auth-provider-inline.version>1.4.0</gravitee-resource-auth-provider-inline.version>
-        <gravitee-resource-auth-provider-ldap.version>1.4.0</gravitee-resource-auth-provider-ldap.version>
+        <gravitee-resource-auth-provider-ldap.version>2.0.0-APIM-7508-secret-support-SNAPSHOT</gravitee-resource-auth-provider-ldap.version>
```

When the version is available, don't forget to change this
```suggestion
        <gravitee-resource-auth-provider-ldap.version>2.0.0</gravitee-resource-auth-provider-ldap.version>
```